bin/rbac-providers-azure

```diff
@@ -25,7 +27,46 @@ from rbac.providers.azure.initial_inbound_sync import initialize_aad_sync
 from rbac.providers.azure.delta_outbound_sync import outbound_sync_listener
 from rbac.providers.azure.delta_inbound_sync import inbound_sync_listener
 
+# LOGGER levels: info, debug, warning, exception, error
+logging.basicConfig(level=logging.INFO)
+LOGGER = logging.getLogger(__name__)
+
+
+class InboundSyncThread(threading.Thread):
+    """Custom Thread subclass that runs an AAD inbound sync listener in its own
+    thread."""
+    def __init__(self):
+        """Initialize the InboundSyncThread class"""
+        threading.Thread.__init__(self)
+        self.name = "AAD Inbound Delta Sync Thread"
+
+    def run(self):
+        """Start the InboundSyncThread"""
+        LOGGER.info("Starting %s", self.name)
+        inbound_sync_listener()
+        LOGGER.info("Exiting %s", self.name)
+
+
+class OutboundSyncThread(threading.Thread):
+    """Custom Thread subclass that runs an AAD outbound sync listener in its own
+    thread."""
+    def __init__(self):
+        """"Initialize the OutboundSyncThread class""""
```

Comment:
Azure daemon is throwing an error because you used four quotes instead of three at the beginning and end of this line.

Reply:
just fixed it